### PR TITLE
Release 0.14

### PR DESCRIPTION
**Pull request by Xophmeister**
See #1651

### CODE REVIEW

**Comment by Xophmeister** [comment]
Demoting, temporarily, to draft status, so #1666 can be reviewed, merged and incorporated into this release...

File: start [hunk 1]
```diff
@@ -203,7 +203,7 @@ http_archive(
     name = "rules_haskell",
     strip_prefix = "rules_haskell-0.14",
     urls = ["https://github.com/tweag/rules_haskell/archive/v0.14.tar.gz"],
-    # sha256 = "...",
+    sha256 = "6a98220d2947e2066949889454320aab5b5cba11d627bfafed03819e91aef2d1",
```
Comment (aherrmann) [inline start:206]: The sha seems to be wrong.
```
$ curl -L https://github.com/tweag/rules_haskell/archive/v0.14.tar.gz|sha256sum
851e16edc7c33b977649d66f2f587071dde178a6e5bcfeca5fe9ebbe81924334  -
```
I can reproduce the same sha locally
```
$ git archive --format=tar.gz --prefix=rules_haskell-0.14/ v0.14|sha256sum
851e16edc7c33b977649d66f2f587071dde178a6e5bcfeca5fe9ebbe81924334  -
```

I think you may have forgotten the `/` at the end of the prefix.
```
$ git archive --format=tar.gz --prefix=rules_haskell-0.14 v0.14|sha256sum
6a98220d2947e2066949889454320aab5b5cba11d627bfafed03819e91aef2d1  -
```

**Review by aherrmann — COMMENTED** [review]
Thank you!
I think there's an issue with the sha.
Apart from that, looks great!

File: start (hunk 1 above)
Comment (Xophmeister) [reply start:206]: Nuts :P OK, let me quickly fix that!

**Review by aherrmann — APPROVED** [review]
Thank you! Good to merge and release! :tada:

**Pull request merged**